### PR TITLE
Port fix entity query in accounts and balances REST apis 

### PR DESCRIPTION
**Pull request by xin-hedera**
**Description**:

This PR ports the entity query fix for accounts and balances REST apis to `main`

 - Union the results from entity table and contract table
 - Add the public key index for the contract table
 
**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-importer/src/main/resources/db/migration/v2/V2.0.5__account_contract_view.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
+-- create a view for accounts and contracts
+create or replace view account_contract as
```
Comment (steven-sheehy) [inline hedera-mirror-importer/src/main/resources/db/migration/v2/V2.0.5__account_contract_view.sql:2]: This should just go at the end of 2.0.0

Comment (Nana-EC) [reply hedera-mirror-importer/src/main/resources/db/migration/v2/V2.0.5__account_contract_view.sql:2]: Correct. Citus isn't released in any environment yet so we can modify V2 migrations for a bit longer.

Comment (xin-hedera) [reply hedera-mirror-importer/src/main/resources/db/migration/v2/V2.0.5__account_contract_view.sql:2]: sure, will move it into 2.0.0

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by ijungmann — APPROVED** [review]

**Pull request merged**